src/common/format.ts

```diff
@@ -86,3 +86,7 @@ export const getParcelOverviewString = (
         (collectionDatetime && ` @ ${dayjs(collectionDatetime).format("DD/MM/YYYY HH:mm")}`)
     );
 };
+
+export const formatTimeStringToHoursAndMinutes = (timeString: string): string => {
```

Comment:
This is to change 10:00:00 into 10:00

Reply:
I think because timeString is HH:mm:ss it becomes invalid date when you put it in new Date(timeString).
I have now used dayjs instead but I need to pass in a 0 otherwise it becomes a single digit.